tools/benchmark_tool/openvino/tools/benchmark/utils/utils.py

```diff
@@ -633,7 +645,9 @@ def get_inputs_info(shape_string, data_shape_string, layout_string, batch_size,
                     reshape = True
                     batch_found = True
                 elif batch_index == -1 and not batch_found and i == len(inputs) - 1:
-                    raise Exception(f"Batch dimension is not specified for this model!")
+                    raise Exception("-b option is provided in command line, but there's no inputs with batch(B) " \
```

Comment:
```suggestion
                    raise RuntimeError("-b option is provided in command line, but there's no inputs with batch(B) " \
```
To align exception type with C++

Reply:
You're right, thank you!